howto/update/upgrade-anbox.md

```diff
@@ -37,39 +62,87 @@ The deployed Juju charms need to be upgraded next.
+[/note]
+
+To upgrade all charms, run the following commands:
+
+    juju refresh --channel=1.17/stable anbox-cloud-dashboard
+    juju refresh --channel=1.17/stable anbox-stream-gateway
+    juju refresh --channel=1.17/stable anbox-stream-agent
+    juju refresh --channel=1.17/stable coturn
+    juju refresh nats
+
+### Upgrade AMS
+
+The AMS service needs to be updated independently of the other service components to ensure minimal down time. The charm can be upgraded by running the following command.
 
-    juju upgrade-charm easyrsa --revision=<rev>
-    juju upgrade-charm etcd --revision=<rev>
-    juju upgrade-charm --channel=1.17/stable lxd
-    juju upgrade-charm --channel=1.17/stable ams
-    juju upgrade-charm --channel=1.17/stable ams-node-controller
-    juju upgrade-charm --channel=1.17/stable aar
+    juju refresh --channel=1.17/stable ams
 
-If you have the streaming stack deployed you have to upgrade also the following charms:
+### Upgrade LXD
 
-    juju upgrade-charm --channel=1.17/stable anbox-stream-gateway
-    juju upgrade-charm --channel=1.17/stable anbox-stream-agent
-    juju upgrade-charm --channel=1.17/stable coturn
-    juju upgrade-charm nats
+As last step you have to upgrade the LXD cluster. Upgrading LXD will cause no restart of running containers but it's recommended to take a backup before continueing.
 
-Once the commands are executed, Juju will perform all necessary upgrade steps automatically.
+As first step you need to upgrade the AMS node controller by running
 
-After Juju has settled the workload status will be marked as `blocked` and the status will show `UA token missing`.
+    juju refresh --channel=1.17/stable ams-node-controller
 
-Anbox Cloud requires a valid Ubuntu Pro token for an Ubuntu Pro subscription which includes the Anbox Cloud entitlement. You can get your Ubuntu Pro token on [Ubuntu Pro](https://ubuntu.com/pro). Please speak with your Canonical account representative.
+Once the upgrade completed, you can continue upgrading LXD:
 
-When you have your Ubuntu Pro token, you can apply it for all relevant charms with the following commands:
+    juju refresh --channel=1.17/stable lxd
 
-    juju config ams ua_token=<your token>
-    juju config lxd ua_token=<your token>
-    juju config ams-node-controller ua_token=<your token>
-    juju config aar ua_token=<your token>
-    juju config anbox-stream-gateway ua_token=<your token>
-    juju config anbox-stream-agent ua_token=<your token>
-    juju config anbox-cloud-dashboard ua_token=<your token>
+In case you want to upgrade individual LXD nodes separately, you can set the following configuration option for the LXD charm before running the refresh command above:
+
+    juju config lxd enable_manual_upgrade=true
+
+This will allow you to run the actual upgrade process for each deployed LXD instance separately. After the charm has been refreshed, the upgrade for a single LXD deployment unit can be triggered by running
```

Comment:
```suggestion
This will allow you to run the actual upgrade process for each deployed LXD instance separately. After the charm has been refreshed, the upgrade for a single LXD deployment unit can be triggered by running:
```